webservices/tasks/download.py

```diff
@@ -141,11 +141,11 @@ def make_bundle(resource):
 
 @app.task(base=QueueOnce, once={"graceful": True})
 def export_query(path, qs):
-    qs = base64.b64decode(qs.encode("UTF-8"))
+    qs = base64.b64decode(qs)
```

Comment:
I removed this because it is redundant. base64decode returns bytes. So no need to encode to bytes https://docs.python.org/3/library/base64.html